composer.json

```
@@ -18,7 +18,7 @@
 		"ext-dom": "*",
 		"masterminds/html5": "^2.7",
 		"php": "^7.2 || ^8.0",
-		"psr/log": "^1.0"
+		"psr/log": "^1.0|^2.0|^3.0"
```

Please let's use this notation (including the spaces) for consistency:

```suggestion
		"psr/log": "^1.0 || ^2.0 || ^3.0"
```